Parser/pegen/pegen.h

```diff
@@ -121,14 +121,15 @@ void *_PyPegen_dedent_token(Parser *p);
 expr_ty _PyPegen_number_token(Parser *p);
 void *_PyPegen_string_token(Parser *p);
 const char *_PyPegen_get_expr_name(expr_ty);
-void *_PyPegen_raise_error(Parser *p, PyObject *, const char *errmsg, ...);
+void *_PyPegen_raise_error(Parser *p, PyObject *, int no_col_number, const char *errmsg, ...);
```

Comment:
Now that you're on it, could you add a name to the second argument (`PyObject *errtype`) as well, please? I added that I think and it now seems a bit weird to have some of the arguments named and others not.

Reply:
Right, will do.